PeriodogramAnalysis/MAIN.py

```diff
@@ -15,26 +18,22 @@
 data: DataLoader = dataset[0]
 
 Periodogram_Analysis = PeriodogramAnalysis(
-        exclude_channels=[ch for ch in range(128) if ch not in [2, 3]],
-        win_sec=4,
-        band=[[0.5, 4], [12, 30]],
-        mark_region = True
+        exclude_channel_list=[ch for ch in range(128) if ch not in [3]],
+        window_length_for_welch=4
     )
 
 Spectrum_Analysis = SpectrumAnalysis(
-        exclude_channels=[ch for ch in range(128) if ch not in [3]],
-        frequency_limit=[0, 10],
-        win_sec=4,
-        nperseg = 100,
-        noverlap= 50,
-        band_display = [0, 2],
-        convert_db = False,
+        exclude_channel_list=[ch for ch in range(128) if ch not in [3]],
+        window_length_for_welch=4,
+        frequency_limit=[0.5, 100],
+        band_display = [0, 5]
     )
 
+bandpass_filter: Operator = ButterBandpass(lowcut=0.2, highcut=200, order=4, tag="bandpass")
```

Comment:
You made the bandpass and never used it...Is the result including the bandpass or not?

Reply:
The result didn't include it, have removed it.